common/config/env/.env

```diff
@@ -1,2 +1,2 @@
 // THIS IS AN AUTO GENERATED FILE BY `RUSH SWITCH <BETA|STABLE>`. DO NOT MODIFY
-COMMUNICATION_REACT_FLAVOR=beta
```

Comment:
Err.... NO.

Reply:
out of interest, this should be gitignored:
![image](https://user-images.githubusercontent.com/2684369/200056569-710ed269-e146-42f6-a1fa-fa9bbfc8c767.png)